packages/dd-trace/src/exporters/log/index.js

```diff
@@ -20,7 +22,7 @@ class LogExporter {
       if (spanStr.length + size + 1 > MAX_SIZE) {
         this._printSpans(queue)
         queue = []
-        size = 0
+        size = TRACE_PREFIX.length + TRACE_SUFFIX.length
```

Comment:
This and the above line should be in some kind of reset function now that there is logic involved since they are duplicated.

Reply:
I've moved this calculation out to a constant. The state is local to the function, so I didn't create a reset method. Also fixed the tests now.